tycho-plugins/repository-utils/src/main/java/org/jboss/tools/tycho/sitegenerator/GenerateRepositoryFacadeMojo.java

```diff
@@ -91,15 +91,18 @@
 	}
 
 	public static final Set<String> defaultSystemProperties = new HashSet<String>(Arrays.asList(new String[] {
+		// these are all parameters of the Jenkins job; if not set they'll be null
 		"BUILD_ALIAS",
 		"JOB_NAME",
 		"BUILD_NUMBER",
 		"BUILD_ID",
-		"HUDSON_SLAVE",
 		"RELEASE",
 		"ZIPSUFFIX",
 		"TARGET_PLATFORM_VERSION",
 		"TARGET_PLATFORM_VERSION_MAXIMUM",
+		
+		// these are environment variables so should be valid when run in Jenkins or for local builds
+		"HOSTNAME", // replaces HUDSON_SLAVE
```

Comment:
Didn't know about NODE_NAME. I'll add that and see if we get a different value than HOSTNAME; nice thing with HOSTNAME is you can set that for local machine even when not running in Jenkins.